src/main/java/org/isf/dicom/manager/SourceFiles.java

```diff
@@ -381,18 +381,12 @@ public static synchronized void loadDicom(FileDicom dicomFileDetail, File source
 				studyUID = dicomObject.getString(Tag.StudyInstanceUID) == null ? studyUID : dicomObject.getString(Tag.StudyInstanceUID);
 				accessionNumber = dicomObject.getString(Tag.AccessionNumber) == null ? accessionNumber : dicomObject.getString(Tag.AccessionNumber);
 				studyDescription = dicomObject.getString(Tag.StudyDescription) == null ? studyDescription : dicomObject.getString(Tag.StudyDescription);
-				//String studyComments = dicomObject.getString(Tag.StudyComments);
 				seriesUID = dicomObject.getString(Tag.SeriesInstanceUID) == null ? seriesUID : dicomObject.getString(Tag.SeriesInstanceUID);
-				//String directoryRecordType = dicomObject.getString(Tag.DirectoryRecordType);
 				seriesInstanceUID = dicomObject.getString(Tag.SeriesInstanceUID) == null ? seriesInstanceUID : dicomObject.getString(Tag.SeriesInstanceUID);
 				seriesNumber = dicomObject.getString(Tag.SeriesNumber) == null ? generateSeriesNumber(patient) : dicomObject.getString(Tag.SeriesNumber);
 				seriesDescriptionCodeSequence = dicomObject.getString(Tag.SeriesDescriptionCodeSequence) == null ?
 						seriesDescriptionCodeSequence :
 						dicomObject.getString(Tag.SeriesDescriptionCodeSequence);
-				//String sliceVector = dicomObject.getString(Tag.SliceVector);
-				//String sliceLocation = dicomObject.getString(Tag.SliceLocation);
-				//String sliceThickness = dicomObject.getString(Tag.SliceThickness);
-				//String sliceProgressionDirection = dicomObject.getString(Tag.SliceProgressionDirection);
```

Comment:
I think it is better to keep these comments as reference on DICOM format

Reply:
They are all simple gets with Tag.XXXX statics as the parameter.   Isn't looking that the Tag class sufficient?  Besides the lines are always available in the git history.

Reply:
You are right!